beancount_dkb/ec.py

```diff
@@ -24,10 +24,12 @@
 
 
 class ECImporter(importer.ImporterProtocol):
-    def __init__(self, iban, account, currency='EUR', file_encoding='utf-8'):
+    def __init__(self, iban, account, currency='EUR', file_encoding='utf-8',
+                 meta_code=None):
```

Comment:
I think you might need to run `black` once on this file to stop the linter from complaining (this should also fix the build).

Reply:
Not sure what `black` is, I use the `autopep` formatter and `flake8` linter in vim. I can look into it as soon as I find the time.

Reply:
Done